spyder/widgets/onecolumntree.py

```diff
@@ -62,15 +62,14 @@ def setup_common_actions(self):
                                      icon=ima.icon('restore'),
                                      triggered=self.restore)
         self.collapse_selection_action = create_action(self,

```

Comment:
@juanis2112 let's transform this into:
```python
self.collapse_selection_action = create_action(
    self,
    text=_('Collapse section'),
    icon=ima.icon('collapse_selection'),
    triggered=self.collapse_selection,
)
```

This makes the code more readable, fixes the pep8speaks issue and also favors changes in the future since we have now an argument per line. (Simpler diffs and less prone to conflicts :-)  )